.github/workflows/main.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           python -m pip install tox
       - name: Run tests
-        run: tox -- --cov-report xml
+        run: tox -e native -- --cov-report xml
```

Comment:
I'd recommend instead to set an environment variable (TOXENV) as that will appear as a separate line, providing a nicer diff (such as from jaraco/skeleton).